### DISCUSSION TITLE
Admin gettin kicked : ERROR: Death Exploit Attempt

### DISCUSSION THREAD

**Discussion by Irethia** [Pryaxis/TShock#1936] [Q&A]
Hi,
I run a litle server for me and my friend and we try to kill the empress of light during the day.
She instakill us (normal) But when she make too much domage we are getting kicked out of the server, even me as a superadmin.
here's some log
2020-05-27 20:49:07 - Commands: INFO: Faerllina authenticated successfully as user: Irethia.
2020-05-27 20:49:14 - Utils: INFO: Imabite executed: /tp t.
2020-05-27 20:49:58 - Utils: INFO: Faerllina executed: /spawnmob 636.
2020-05-27 20:50:24 - TShock: INFO: Faerllina disconnected.
2020-05-27 20:50:24 - TSPlayer: INFO: Kicked Faerllina for : 'Failed to shade polygon normals.'
2020-05-27 20:50:24 - TextLog: ERROR: Death Exploit Attempt: Damage 28112
2020-05-27 20:50:25 - TShock: INFO: Trichou disconnected.
2020-05-27 20:50:25 - TSPlayer: INFO: Kicked Trichou for : 'Failed to shade polygon normals.'
2020-05-27 20:50:25 - TextLog: ERROR: Death Exploit Attempt: Damage 30839

Is there some command lines or anything else to prevent that
Thanks and have a good day

**Comment by moisterrific** [comment]
There’s nothing to prevent that, apparently this was added in bouncer to prevent server crash exploits. IIRC if damage taken >20,000 it will force kick you.

**Reply by moisterrific** [reply]
![8D7A1C25-806B-4F64-B3E3-443F1079DC91](https://user-images.githubusercontent.com/57187883/83144771-db80b900-a0c1-11ea-8ac4-9c2260737adf.png)

**Reply by Ayrawei** [reply]
I was kicked by the same bug. I understand it is there for exploit protection, but would it not be possible to add an exception for Empress of Light since her giving you 20 000+ damage during enrage is perfectly legit and not a death exploit attempt?

**Reply by moisterrific** [reply]
I don’t know exactly why the threshold is set to >20k damage. Maybe it can be raised to accommodate for day Empress instakill damage? Or would that greatly compromise server stability? Need TShock devs to weigh in on this.

**Reply by hakusaro** [reply]
@moisterrific what's the damage cap?

**Reply by moisterrific** [reply]
The damage cap of Empress? Not sure the exact maximum amount, but for me it ranged from 15-25k.

**Comment by hakusaro** [comment]
Based on the feedback from @Irethia, @Ayrawei, & @moisterrific in this thread (in particular the damage logs from @Irethia) I've bumped the default damage cap to 42,000. This should be available in an upcoming release (or the latest build, if you prefer). Please let me know if this is not enough after that change is widely available.

**Reply by moisterrific** [reply]
Is there any specific reason it was set to 20k to begin with? Like higher than that would crash the server or something?

**Reply by hakusaro** [reply]
Well the comment says it causes infinite loops in the server, so I'm guessing that it was some old bug. It may still exist, but I'm not sure if it does.
